blockchain/vm/jump_table.go

```diff
@@ -71,7 +71,7 @@ type JumpTable [256]operation
 
 func newIstanbulInstructionSet() JumpTable {
 	instructionSet := newConstantinopleInstructionSet()
-	// TODO-IstanbulCompatible: add istanbul hardfork items
+	EnableEIP(1344, &instructionSet)
```

Comment:
An error is not handled here.

Reply:
@yoomee1313 We can call `enable1344` directly as geth do.

Reply:
@ehnuje @aidan-kwon thanks! i edited to call `enable1344`. https://github.com/klaytn/klaytn/pull/952/commits/4bbb9f2c6b504e74357d42d343be196b952abdee.